pages/learn/integration/pre-built-integrations/ftp-edi-message-to-salesforce-opportunity.js

```diff
@@ -315,15 +315,15 @@ export default function Learn({ samples, content }) {
                   <Col xs={12} lg={6} style={{ fontSize: "18px" }}>
                     <p>Interactions between businesses, such as sending purchase orders and invoices, usually occur over
                       EDI-based B2B channels. On the other hand, sales and customer details related to such transactions
-                      are maintained in Salesforce. Therefore, it&apos;s critical to bridge salesforce with B2B channels in
-                      order to automate the sales processes. For example, it is It is possible to update the status and
-                      details of opportunities in Salesforce based on the exchange of EDI messages like EDIFACT REQOTE
-                      (Request for quotation), EDIFACT QUOTES (Quotation response), EDIFACT ORDERS (Purchase order), etc.
+                      are maintained in Salesforce. Therefore, it&apos;s critical to bridge Salesforce with B2B channels in
+                      order to automate the sales processes. For example, it is possible to update the status and
+                      details of opportunities in Salesforce based on the exchange of EDI messages like `EDIFACT REQOTE`
+                      (Request for quotation), `EDIFACT QUOTES` (Quotation response), `EDIFACT ORDERS` (Purchase order), etc.
                       Such integrations will eliminate the delays and inconsistencies in updating Salesforce and provide
-                      up-to-date information for sales staff and decision-makers.
+                      up-to-date information for sales staff and decision makers.
                     </p>
 
-                    <p>The code sample below reads EDIFACT REQOTE EDI files from a given FTP location and creates a Salesforce Opportunity.
+                    <p>The code sample below reads `EDIFACT REQOTE` EDI files from a given FTP location and creates a Salesforce opportunity.
```

Comment:
Can't we use capital here as opportunity is an object name?

Reply:
Fixed.